composition-js/src/__tests__/validation_errors.test.ts

```diff
@@ -112,3 +112,57 @@ describe('@requires', () => {
     ]);
   });
 });
+
+
```

Comment:
In the comments of validation.ts you have a union as an example, did you want to have that as an explicit test as well or are the interface / union examples sufficently similar?

Reply:
You're right, it's worth having both and pulled the example from `validation.ts` into a proper test (which also ensure the comment is actually correct, which it wasn't cause I had forgotten some `@shareable` :)).